lib/src/screens/detail_screen/detail_screen.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:syncfusion_flutter_charts/sparkcharts.dart';
+
+import '../../constants/colors.dart';
+
+class DetailScreen extends StatefulWidget {
+  @override
+  _DetailScreenState createState() => _DetailScreenState();
+}
+
+//Class which is used as type for the data source.
+
+class _DetailScreenState extends State<DetailScreen> {
+  @override
+  void initState() {
+    // Calling getData method to add data to the list
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: ColorsApp.backgroundColor,
+      appBar: AppBar(
+        elevation: 0,
+        backwardsCompatibility: false,
+        systemOverlayStyle:
+            SystemUiOverlayStyle(statusBarColor: Colors.transparent),
+        backgroundColor: Colors.transparent,
+        centerTitle: true,
+        leading: IconButton(
+          icon: Icon(
+            Icons.arrow_back,
+            size: 40,
+          ),
+          onPressed: () {
+            Navigator.pop(context);
```

Comment:
should add canPop method to make sure that your screen can be pop

Reply:
fixed